entity-framework/core/providers/sqlite/functions.md

```diff
@@ -97,29 +97,79 @@ dateTime.Year                   | strftime('%Y', @dateTime)
+double.DegreesToRadians(degrees) | radians(@degrees)                     | EF Core 8.0
+double.RadiansToDegrees(radians) | degrees(@dradians)                    | EF Core 8.0
+doubleValue % d                  | mod(@doubleValue, @d)
+EF.Functions.Random()            | abs(random() / 9223372036854780000.0)
+float.DegreesToRadians(degrees)  | radians(@degrees)                     | EF Core 8.0
+float.RadiansToDegrees(radians)  | degrees(@radians)                     | EF Core 8.0
+floatValue % d                   | mod(@floatValue, @d)
+Math.Abs(value)                  | abs(@value)
+Math.Acos(value)                 | acos(@value)                          | EF Core 8.0
+Math.Acosh(d)                    | acosh(@d)                             | EF Core 8.0
+Math.Asin(d)                     | asin(@d)                              | EF Core 8.0
+Math.Asinh(d)                    | asinh(@d)                             | EF Core 8.0
+Math.Atan(d)                     | atan(@d)                              | EF Core 8.0
+Math.Atan2(y, x)                 | atan2(@y, @x)                         | EF Core 8.0
+Math.Atanh(d)                    | atanh(@d)                             | EF Core 8.0
+Math.Ceiling(d)                  | ceiling(@d)                           | EF Core 8.0
+Math.Cos(d)                      | cos(@d)                               | EF Core 8.0
+Math.Cosh(value)                 | cosh(@value)                          | EF Core 8.0
+Math.Exp(d)                      | exp(@d)                               | EF Core 8.0
+Math.Floor(d)                    | floor(@d)                             | EF Core 8.0
+Math.Log(d)                      | ln(@d)                                | EF Core 8.0
+Math.Log(a, newBase)             | log(@newBase, @a)                     | EF Core 8.0
+Math.Log2(x)                     | log2(@x)                              | EF Core 8.0
+Math.Log10(d)                    | log10(@d)                             | EF Core 8.0
+Math.Max(val1, val2)             | max(@val1, @val2)
+Math.Min(val1, val2)             | min(@val1, @val2)
+Math.Pow(x, y)                   | pow(@x, @y)                           | EF Core 8.0
+Math.Round(d)                    | round(@d)
+Math.Round(d, digits)            | round(@d, @digits)
+Math.Sign(d)                     | sign(@d)                              | EF Core 8.0
+Math.Sin(a)                      | sin(@a)                               | EF Core 8.0
+Math.Sinh(value)                 | sinh(@value)                          | EF Core 8.0
+Math.Sqrt(d)                     | sqrt(@d)                              | EF Core 8.0
+Math.Tan(a)                      | tan(@a)                               | EF Core 8.0
+Math.Tanh(value)                 | tanh(@value)                          | EF Core 8.0
+Math.Truncate(d)                 | trunc(@d)                             | EF Core 8.0
+MathF.Acos(x)                    | acos(@x)                              | EF Core 8.0
```

Comment:
Maybe it's worth just adding an asterisk and a footnote, saying that all corresponding MathF methods are translated as well, to not make this needlessly long.

Reply:
Yeah, I like it. I was also going to add a note that corresponding generic math methods are also translated, so I'll do both at the same time.